xbmc/music/MusicDatabase.cpp

```diff
@@ -4818,6 +4818,7 @@ bool CMusicDatabase::GetDiscsByWhere(CMusicDbUrl& musicUrl,
         pItem->SetLabel2(record->at(0).get_asString()); // GUI show label2 for disc sort order??
         pItem->GetMusicInfoTag()->SetDiscNumber(
```

Comment:
Since a one line change, could you tidy up this previous lines too. Remove comment and thus SetDiscNumber on one line.